container/container_test.go

```diff
@@ -21,18 +21,27 @@ func newTesting(t *testing.T) (*DockerContainer, *mocks.CommonAPIClient) {
 
 	c, err := New(ClientOption(m))
 	require.NoError(t, err)
+	require.NotZero(t, c)
 
 	return c, m
 }
 
 func mockNew(m *mocks.CommonAPIClient) {
-	m.On("NegotiateAPIVersion", mock.Anything).Once().Return()
-	m.On("Info", mock.Anything).Once().Return(types.Info{Swarm: swarm.Info{NodeID: "1"}}, nil)
-	m.On("NetworkInspect", mock.Anything, "core", types.NetworkInspectOptions{}).Once().
+	m.On("NegotiateAPIVersion", mock.Anything).
+		Once().
+		Return()
```

Comment:
I like this pattern because when you have a few _m.On()_ definitions, it's easier to read them and understand their behavior this way. I did it by intention.

Reply:
I agree with @krhubert. This is not go style.

Reply:
It's not like when you will have 5 chain calls it must be in single funciton, jsut if it can be write in one line let's do this.